### DISCUSSION TITLE
Ch3nyang's blog | SEEDLabs (5) - SQL injection

### DISCUSSION THREAD

**Discussion by giscus** [WCY-dt/wcy-dt.github.io#22] [Comments]
# Ch3nyang's blog | SQL injection

本文为 SEED Labs 2.0 - SQL Injection Attack Lab 的实验记录。...

https://blog.ch3nyang.top/post/seedlab_SQL/

**Comment by futurehqy-lj** [comment]
我想问问http://www.seed-server.com/这个网址火狐进不去怎么办，

**Reply by WCY-dt** [reply]
这个和火狐无关，你需要把 docker 运行起来后才能进去。

如果还进不去，建议检查一下 docker 的设置，是不是不小心把端口给关闭了。

**Reply by futurehqy-lj** [reply]
谢谢up主！！！！但是我的页面一直显示Apache2 Ubuntu Default Page默认页面，dcup警告：AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 10.9.0.5. Set the 'ServerName' directive globally to suppress this message
网上说的修改apache的配置文件我也试过了，severname10.9.0.5，重启apache也还是这样。

**Reply by WCY-dt** [reply]
不好意思，我试了一下，没法复现这个问题，也不知道这个问题出在哪里。

要不重装一下环境吧。按照我的经验，有时候前一个lab调了一些设置忘了改，可能会影响到其它lab😂
